[2018-06-17 09:01:46] logger_core.ERROR: Не удалось полключиться к базе данных [] []
[2018-06-17 09:01:46] logger_core.ERROR: Не удалось полключиться к базе данных [] []
